packages/core/src/routes/admin-user/basics.ts

```diff
@@ -111,13 +116,29 @@ export default function adminUserBasicsRoutes<T extends AuthedRouter>(...args: R
         primaryEmail: string().regex(emailRegEx),
         username: string().regex(usernameRegEx),
         password: string().min(1),
+        passwordEncrypted: string(),
+        passwordEncryptionMethod: nativeEnum(UsersPasswordEncryptionMethod),
```

Comment:
not sure whether we should check whether the encrypted password is a valid encryption of specified encryption method. For example, `passwordEncrypted` could contains invalid characters which are not supported by the encryption method.

Reply:
We are unable to check the correction because we do not know the plan text password. So no additional check is necessary I think.